dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityInstanceQueryParams.java

```diff
@@ -1268,6 +1272,11 @@ public static boolean isStaticColumn( String propName )
                 .anyMatch( orderColumn -> orderColumn.getPropName().equals( propName ) );
         }
 
+        public static boolean isFieldEqualToEnrolledAt( String fieldName )
+        {
+            return ENROLLED_AT.getPropName().equalsIgnoreCase( fieldName );
+        }
+
```

Comment:
A method with the name `isPropertyEqualTo`  and then it only compares ENROLLED_AT doesn't make much sense to me